.github/workflows/transform-tokens.yml

```diff
@@ -0,0 +1,27 @@
+name: Transform tokens using token-transformer
+
+on:
+  push:
```

Comment:
So now that we're generating the `transformed-tokens`, I'm assuming after this MR we'll change the Style dictionary job to only be triggered on the success of this job?

Reply:
Excellent, you've got it covered!